packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -113,14 +113,26 @@ class _LayoutBuilderElement extends RenderObjectElement {
           built = widget.builder(this, constraints);
           debugWidgetBuilderValue(widget, built);
         } catch (e, stack) {
-          built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack));
+          final Element element = this;
+          built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack,
+            informationCollector:() sync* {
+              yield element.describeOwnershipChain('Elements chain');
+              yield element.describeFirstLocalWidget('Local Widget');
+            }
+          ));
         }
       }
       try {
         _child = updateChild(_child, built, null);
         assert(_child != null);
       } catch (e, stack) {
-        built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack));
+        final Element element = this;
+        built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack,
```

Comment:
nit: follow flutter style for indentation.

Reply:
fixed